src/nxdoc/nuxeo-server/administration/variables-and-configuration/variables-and-configuration-pattern.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Variables and Configuration Pattern
+review:
+    comment: ''
+    date: '2020-12-14'
+    status: ok
+tree_item_index: 90
+toc: true
+---
+
+{{! excerpt}}
+This page provides clarifies all options available in the current Nuxeo framework.
+{{! /excerpt}}
+Variable Options
+
+## Glossary
+
+Let's review first the main concepts:
+- **Environment variables**: variables set in the OS that cannot be changed at runtime
+- **System properties**: variables set on the Java command line using the `-Dpropertyname=value` syntax, that can be added at runtime using `System#setProperties` or `System.getProperties#load` method, retrieved through the various `System#getProperty` methods.
+- **Nuxeo configuration properties**: variables set in the [nuxeo.conf]({{page version='' space='nxdoc' page='configuration-parameters-index-nuxeoconf'}}) file.
```

Comment:
Maybe it's ok at doc build but for info, the "space" attribute (unneeded here) breaks the link in my preview